src/components/Menu/styles.ts

```diff
@@ -11,13 +11,20 @@ export default createUseStyles(
     onlyCustomContent?: boolean
   ) => ({
     overlay: {
-      width: '100%',
-      height: '100%',
+      position: 'absolute',
+      top: -100,
+      bottom: -100,
+      left: -100,
+      right: -100,
+      zIndex: 100,
```

Comment:
Not sure if those changes broke the menu/select component. In your PR closing menu stopped working. So, if you open the menu by clicking on the trigger (combobox for the select), you are not able to close it by trigger click again, only by outside click.